MoltenVK/MoltenVK/GPUObjects/MVKPipeline.h

```diff
@@ -430,6 +430,8 @@ class MVKGraphicsPipeline : public MVKPipeline {
 	uint32_t _tessCtlPatchOutputBufferIndex = 0;
 	uint32_t _tessCtlLevelBufferIndex = 0;
 
+	static constexpr uint32_t _maxVertexInputBindingBufferCount = 31u; // Taken from Metal Feature Set Table. Highest value out of all present GPUs
```

Comment:
The naming pattern for all `static constexpr` is (or should be) `kMVK...`. See top of `MVKDevice.h` for other examples. Suggest making this `kMVKMaxVertexInputBindingBufferCount`.

Reply:
```suggestion
	static constexpr uint32_t kMVKMaxVertexInputBindingBufferCount = 31u; // Taken from Metal Feature Set Table. Highest value out of all present GPUs
```